admin.js

```diff
@@ -109,20 +109,23 @@
     helpHook = function (moduleName, origionalHelp, config) {
```

Comment:
Help hook probably won't work on all modules anymore, because the help method doesn't always exist and might not be generated (pulled directly out of `__descriptor.help` rather than generating a function that returns it)